contents/product-engineers/nps-vs-csat-vs-ces.md

```diff
@@ -0,0 +1,258 @@
+
+| Product | NPS score |
+|--------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------|
+| [Snowflake](https://www.snowflake.com/blog/customer-experience-report-2022/) | 72 |
+| [Square](https://d1g145x70srn7h.cloudfront.net/documents/investor-relations/presentations/05-11-2018-overview.pdf) | 70 |
+| [Zoom](https://blog.zoom.us/zooms-nps-dominates-video-conferencing/) | 69 | 
+| [DocuSign](https://www.docusign.com/why-docusign) | 66 |
+| [Zapier](https://www.comparably.com/brands/zapier) | 64 |
+| [Slack](https://www.comparably.com/brands/slack) | 53 |
+| [Mailchimp](https://www.comparably.com/brands/mailchimp) | 47 |
+| [Netflix](https://www.comparably.com/brands/netflix) | 46 |
+| [Twilio](https://www.comparably.com/brands/twilio) | 34 |
+| [Dropbox](https://www.comparably.com/brands/dropbox) | 27 | 
+
+### Drawbacks of NPS
+
+NPS can be tricky because it's often contextless. It gives you an indication of your users' sentiment toward your product but not much else.
+
+To counteract this, it's best to ask follow-up questions in your NPS survey, like:
+
+- Which features do you find most useful?
+- What are you finding difficult to do?
+- Is there a feature or aspect of the product that particularly disappoints you?
+- What would it take to improve your score to a 9 or 10?
+- Would you be open to a follow-up call to discuss your feedback in more detail?
+
+NPS is particularly handy for understanding long-term user satisfaction.
+
+With this in mind, it's best to wait for users to properly experience your app. Then, you should run NPS surveys every month or quarter to understand how their experience changes over time. This enables you to identify trends in how your product changes are affecting your user experience.
+
+### How to use NPS to improve your app
+
+First, focus on the feedback from your main target audience only. Do this by assigning a [user persona](/product-engineers/how-to-create-user-personas) to each person that responded to your survey and then filtering out those who don't match your key persona.
+
+Next, **focus your attention on the passive responses** (the users who responded with 7 or 8). These users will be the easiest to convert into active promoters of your product and there's usually just one or two things holding them back.
+
+Find out what these things are (by conducting interviews or follow-up surveys) and address their feedback with product changes. Once you've done so, let them know and they'll be grateful you've taken their input seriously. This should persuade them into becoming promoters.
+
+Once you've done so, repeat the same steps for detractors who responded with 5 or 6. You can also repeat the process again for detractors who responded from 0-4, but keep in mind that the lower their response is, the more effort it will take to improve – which may not be worthwhile.
```

Comment:
```suggestion
Then, repeat the same steps for detractors who responded with 5 or 6. You can also repeat the process again for detractors who responded from 0-4, but keep in mind that the lower their response is, the more effort it will take to improve – which may not be worthwhile.
```